cves/kernel/CVE-2022-29156.yml

```diff
@@ -471,11 +519,11 @@ CWE_instructions: |
 CWE:
 - 415
 CWE_note: |
-  CWE as registered in the NVD. If you are curating, check that this
-  is correct and replace this comment with "Manually confirmed".
+   Manually confirmed
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
   If the report mentions a nickname, use that.
   Must be under 30 characters. Optional.
-nickname:
+nickname: |
```

Comment:
This nickname doesn't really work because double free vulnerabilities are a type of vulnerability.

Reply:
I can see how that could cause confusion. I will change that